distribution.pm

```diff
@@ -240,6 +241,14 @@ sub script_output {
     return $out;
 }
 
+sub set_serial_failures {
```

Comment:
POD? ^^

Reply:
yes, doc would be appreciated. Also I wonder if there can not be a better name. How about `set_expected_serial_failures`?